### ISSUE TITLE
Rails 4 compatibility

### ISSUE THREAD

**Issue by rutgerg** [DavyJonesLocker/client_side_validations-simple_form#31]
Any plans of making the gem rails 4 compatible?

    Bundler could not find compatible versions for gem "actionpack":
      In Gemfile:
        client_side_validations-simple_form (>= 0) ruby depends on
          actionpack (~> 3.0) ruby

        rails (= 4.0.0) ruby depends on
          actionpack (4.0.0)

**Comment by spidie** [comment]
We have managed to have this working before with the 4-0-beta version of client_side_validations on Rails 4.0.0 - it worked because the spec for 4-0-beta actually says to 3.2 so  client_side_validations-simple_form was able to resolve dependency.

Now we've upgraded to 4.2 this is now broken as the 4-2-stable version of client_side_validations not supported by client_side_validations-simple_form. 

I might fork the project today and see if a simple version dependency change might fix it - otherwise I might have to say +1 as I probably don't understand the internals of this gem enough to be able to hack on it.
